### ISSUE TITLE
res auto

### ISSUE THREAD

**Issue by dejwiix** [Zrips/Residence#721]
i write command /res auto and them write area size is not within your allowed limits

**Comment by pavelsx** [comment]
Yes...same problem...i write command /res auto and them write area size is not within your allowed limits but i dont have limits :D

**Comment by DrakMC** [comment]
I have the same problem.

**Comment by BatlleFury** [comment]
I have this problem too...

**Comment by XC012** [comment]
+1

**Comment by Zrips** [comment]
7e201f517716563b2d40dedd752f19d461d960b5
